sensor/common/collectorruntimeconfig/handler.go

```diff
@@ -0,0 +1,123 @@
+package collectorruntimeconfig
```

Comment:
I am not sure if it is better for `collectorruntimeconfig` to have its own handler or if it should be added to the handler for `networkflows`.

Reply:
I think separate handlers are probably needed, because certain handler methods such as `SensorCapability` should be different for collector config and CIDR blocks. Another advantage of having separate handlers is that code that does different things is not mixed together.

I think the advantage of having one handler is reducing repetition of code.

Reply:
Thanks for explaining, that sounds good.